ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/copy_to_clipboard.js

```diff
@@ -0,0 +1,137 @@
+const copyStyle = `
+.code-n-btn-wrapper {
+  positon: relative;
+}
+.copy-code-wrapper {
+  display: none;
+  position: absolute;
+  right: 0;
+  margin: 10px 15px 0 0;
+}
```

Comment:
Once " .code-n-btn-wrapper" is fixed, please use top/right to position the button and remove the margin (which has a hidden dependency to the current theme).

Reply:
Ok